src/rust/storage/seg/src/ttl_buckets/ttl_buckets.rs

```diff
@@ -125,6 +125,19 @@ impl TtlBuckets {
         EXPIRE_TIME.add(duration.as_nanos() as _);
         expired
     }
+
+    pub(crate) fn clear(&mut self, hashtable: &mut HashTable, segments: &mut Segments) -> usize {
+        let start = Instant::now();
+        let mut cleared = 0;
+        for bucket in self.buckets.iter_mut() {
+            cleared += bucket.clear(hashtable, segments);
+        }
+        segments.set_flush_at(CoarseInstant::now());
```

Comment:
Considering the semantics of the memcached `flush_all` command, which allows delayed flushing, I wonder if we should set this outside this function. This way, we can set that `flush_at` in segments when we receive the command, and let the storage module handle the actual flushing internally (later).